src/test/java/wooteco/subway/domain/StationTest.java

```diff
@@ -1,16 +1,27 @@
 package wooteco.subway.domain;
 
-import org.assertj.core.api.Assertions;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 class StationTest {
 
+    private static final String EXCESS_MAX_LENGTH_STRING = "-".repeat(256);
+
     @DisplayName("지하철역의 이름이 공백인지를 검사한다.")
     @Test
     public void blankNameTest() {
         // given & when & then
-        Assertions.assertThatThrownBy(() -> new Station(""))
+        assertThatThrownBy(() -> new Station(""))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("지하철역은 255보다 크거나 같을 수 없다.")
```

Comment:
```suggestion
    @DisplayName("지하철역의 이름은 255보다 크거나 같을 수 없다.")
```

그런데 크거나 같을 수 없다면 "같은" 상황도 검증해 봐야 하는 것 아닐까요?

Reply:
255자 까지는 가능하고 256자부터 불가능하므로 "지하철역의 이름은 255보다 클 수 없다."로 수정하였습니다!